api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -297,4 +297,25 @@ default AppendFiles newFastAppend() {
    * Returns a {@link LocationProvider} to provide locations for new data files.
    */
   LocationProvider locationProvider();
+
+  /**
+   * Returns the current refs for the table
+   *
+   * @return the current refs for the table
+   */
+  Map<String, SnapshotRef> refs();
```

Comment:
Shouldn't this also have a default that creates a map of "main" to the current snapshot ID?

Reply:
Yeah, that sounds reasonable as well.